.github/workflows/testing-and-deployment.yml

```diff
@@ -41,7 +41,6 @@ jobs:
         run: |
           pip install .
           python -c "import pyvista;print(pyvista.Report(gpu=False));from pyvista import examples;print('User data path:', examples.USER_DATA_PATH)"
-
```

Comment:
These line breaks are outside the scope of this PR. Let's change this in a follow-up and discuss if we want to standardize this. If we do, let's create a pre-commit for it.